vagrant/boxes.d/02-kubevirt.yaml

```diff
@@ -0,0 +1,26 @@
+centos7-devel-foreman-kubevirt:
```

Comment:
It's a katello devel playbook so `centos7-devel-katello-kubevirt`?

Reply:
this box will go away, only the infra box will remain